js/modules/k6/html/gen/gen_elements.go

```diff
@@ -26,204 +27,203 @@ type ElemInfoCollectorState struct {
-	"Legend AccessKey string",
-	"Li Value int=0",
-	"Li Type enum=,1,a,A,i,I,disc,square,circle",
-	"Link CrossOrigin enum-nullable=anonymous,use-credentials",
-	"Link ReferrerPolicy enum=,no-referrer,no-referrer-when-downgrade,origin,origin-when-cross-origin,unsafe-url",
-	"Link Href url-or-blank",
-	"Link Hreflang string",
-	"Link Media string",
-	"Link Rel string",
-	"Link Target string",
-	"Link Type string",
-	"Map Name string",
-	"Meta Content string",
-	"Meta Name string",
-	"Meta HttpEquiv=http-equiv enum=content-type,default-style,refresh",
-	"Meter Min int",
-	"Meter Max int",
-	"Meter High int",
-	"Meter Low int",
-	"Meter Optimum int",
-	"Mod Cite string",
-	"Mod Datetime string",
-	"Object Data url-or-blank",
-	"Object Height string",
-	"Object Name string",
-	"Object Type string",
-	"Object TabIndex int=0",
-	"Object TypeMustMatch bool",
-	"Object UseMap string",
-	"Object Width string",
-	"OList Reversed bool",
-	"OList Start int",
-	"OList Type enum=1,a,A,i,I",
-	"OptGroup Disabled bool",
-	"OptGroup Label string",
-	"Option DefaultSelected=selected bool",
-	"Option Selected bool",
-	"Output HtmlFor=for string",
-	"Output Name string",
-	"Output Type const=output",
-	"Param Name string",
-	"Param Value string",
-	"Pre Name string",
-	"Pre Value string",
-	"Quote Cite string",
-	"Script CrossOrigin string",
-	"Script Type string",
-	"Script Src url-or-blank",
-	"Script Charset string",
-	"Script Async bool",
-	"Script Defer bool",
-	"Script NoModule bool",
-	"Select Autofocus bool",
-	"Select Disabled bool",
-	"Select Multiple bool",
-	"Select Name string",
-	"Select Required bool",
-	"Select TabIndex int",
-	"Source KeySystem string",
-	"Source Media string",
-	"Source Sizes string",
-	"Source Src url-or-blank",
-	"Source Srcset string",
-	"Source Type string",
-	"Style Media string",
-	"Table Sortable bool",
-	"TableCell ColSpan int=1",
-	"TableCell RowSpan int=1",
-	"TableCell Headers string",
-	"TableHeaderCell Abbr string",
-	"TableHeaderCell Scope enum=,row,col,colgroup,rowgroup",
-	"TableHeaderCell Sorted bool",
-	"TextArea Type const=textarea",
-	"TextArea Value string",
-	"TextArea DefaultValue=value string",
-	"TextArea Placeholder string",
-	"TextArea Rows int",
-	"TextArea Cols int",
-	"TextArea MaxLength int",
-	"TextArea TabIndex int",
-	"TextArea AccessKey string",
-	"TextArea ReadOnly bool",
-	"TextArea Required bool",
-	"TextArea Autocomplete enum=on,off",
-	"TextArea Autocapitalize enum=sentences,none,off,characters,words",
-	"TextArea Wrap enum=soft,hard,off",
-	"Time Datetime string",
-	"Track Kind enum=subtitle,captions,descriptions,chapters,metadata",
-	"Track Src url-or-blank",
-	"Track Srclang string",
-	"Track Label string",
-	"Track Default bool",
-	"UList Type string",
+// "Elem" is the struct name for an Element. "Method" is the go method name. "Attr" is the name of the DOM attribute the method will access.
+// "TemplateType" is used the "elemFuncsTemplate" and the returnType function - either string
```

Comment:
Can we make TemplateType a bunch of constants? I'm not entirely happy with all these repeated strings.